src/components/Sidebar/Element/Element.ts

```diff
@@ -36,6 +46,7 @@ export class CandySidebarElement extends LitElement {
           ? "element-container-extended"
           : null}"
         ?disabled="${this.disabled}"
+        @click="${this.handleClick}"
       >
         <slot name="icon"></slot>
         <p>${!this.collapsed ? this.label : null}</p>
```

Comment:
Can you add collapse condition like this one just below to cover end-icons element?
It should be hidden when in collapse state

It has been done in the accordion component check it there if you need example